src/workspace/Lesson/Section/Editor.jsx

```diff
@@ -126,13 +93,12 @@ export default Editor
 
 function PlainEditorContainerInner(props) {
 	const dispatch = useScopedDispatch()
-	const hasPendingChanges = useScopedSelector(hasPendingChangesSelector())
-	const [editable, setEditable] = React.useState(
-		props.editable === undefined ? true : props.editable
-	)
+	//	const hasPendingChanges = useScopedSelector(hasPendingChangesSelector())
+	//	const [editable, setEditable] = React.useState(
+	//		props.editable === undefined ? true : props.editable
+	//	)
```

Comment:
could be removed?

Reply:
I do not know is you code, i can remove it.